tiledb/api/examples/using_tiledb_stats.rs

```diff
@@ -0,0 +1,117 @@
+extern crate tiledb;
+use tiledb::{Array, Result as TileDBResult};
+use tiledb::config::Config;
+use tiledb::vfs::VFS;
+use tiledb::Datatype;
+
+
+const ARRAY_NAME: &str = "stats_array";
+const ATTRIBUTE_NAME: &str = "a";
+
+pub fn create_array(row_tile_extent : u32, col_tile_extent : u32) -> TileDBResult<()> {
```

Comment:
I see that the C example doesn't have them but I do think examples should be well-commented, at least to say what the array we're working on looks like.

(I see I did not follow my own advice in the other example.. I will fix that)

Reply:
#34 was merged to `main` so now I have followed my own advice :)

Reply:
Did some docs for the example!